packages/react/src/components/ListItem/index.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import { Icon } from "..";
+import { ICONS } from "../../constants/";
+
+const ListItem = ({
+  active,
+  title,
+  subtitle,
+  subitem,
+  miniitem,
+  value,
+  leftIcon,
+  rightIcon,
+  hideDivider,
+  onClick,
+}) => (
+  <div
+    onClick={onClick}
+    className={cx({
+      ListItem: true,
```

Comment:
You can move this in the `cx` declaration

`cx("ListItem", {`

Reply:
👍